image_segmentation:
  ros__parameters:
    update_rate: 30.0

    rgb_topic: "/k4a/rgb/image_raw"
    depth_topic: "/k4a/depth_to_rgb/image_raw"

    hsv_hue_low: 0
    hsv_hue_high: 179
    hsv_saturation_low: 0
    hsv_saturation_high: 255
    hsv_value_low: 0
    hsv_value_high: 100

    depth_low: 200
    depth_high: 430
    depth_scale: 1.0

    connectivity: 4

point_cloud_generator:
  ros__parameters:
    update_rate: 10.0

    depth_scale: 1000.0                 
    voxel_size: 0.002                    

    tf_x: 0.027
    tf_y: 0.033
    tf_z: 0.137
    tf_qx: 0.0
    tf_qy: 0.0
    tf_qz: 0.0
    tf_qw: 1.0

point_clouds_alignment_scaling:
  ros__parameters:
    update_rate: 10.0

    voxel_size: 0.002
    alignment_percentile: 98.0